python/send_message/main.py

```diff
@@ -0,0 +1,36 @@
+from .functions.send_sms_twilio import send_sms_twilio
+from .functions.send_email_mailgun import send_email_mailgun
+from .functions.send_tweet import send_tweet
+from .functions.send_message_discord_webhook import send_message_discord_webhook
+
+
+def main(req, res):
+    """Main Function"""
+
+    global result
```

Comment:
It's probably not necessary to use `global`:

```suggestion
   result = {}
```